.github/workflows/release.yml

```
@@ -57,28 +130,38 @@ jobs:
         with:
           name: contracts
           path: |
-            contracts/target/wasm32-unknown-unknown/release/*.wasm
+            contracts/account/target/wasm32-unknown-unknown/release/*.wasm
 
+  # ── Create GitHub release ────────────────────────────────────────────────
   github-release:
     name: Create GitHub Release
     needs: [build, contracts]
     runs-on: ubuntu-latest
+    if: env.DRY_RUN == 'false'
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)